DB/classification/models.py

```diff
@@ -57,6 +52,32 @@ def __init__(self, model_type="default", created_at=datetime.datetime.utcnow(),
         self.model = model
         self.id = _id
 
+    @classmethod
+    def reload_db_info(cls):
+        """
+        :summary: Reloads which database/user pair to use (for testing). Defaults to seads/seadapi when no file is present (old behavior)
+        """
+        db_info = None
+        try:
+            db_info = open("db_info", 'r')
```

Comment:
Most of the functionality of this package is duplicated and unnecessarily complicated connection logic. If possible, try to use the same connection as the rest of the API.

Now that is out of the way, the rest of it can be tested pretty easily. Replace the appropriate query functions with stubs that check that they got the expected query and return the expected data. It may be helpful the abstract the database driver library and then stub that out instead.